src/main/webapp/package.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
This file should be at the root of repo

Reply:
Will react see package.json file if it's at the root?